django_rq/queues.py

```diff
@@ -72,21 +72,29 @@ def get_redis_connection(config):
         else:
             # We're using django-redis-cache
             return cache._client
-        
+
```

Comment:
You know, I thought about this approach when I originally created this PR. I originally didn't do it this way because I (incorrectly) assumed that `StrictRedis` did not have a `from_url` method. It appears I was wrong-- I'll make this change, it's much cleaner.